service/tasks/__init__.py

```diff
@@ -1,7 +1,7 @@
 import service.tasks.accounts
 import service.tasks.monitoring
 import service.tasks.driver
-import service.tasks.volume
+# import service.tasks.volume
```

Comment:
You are right. But we might need `volume` later after the summit. So, currently, we just disable it.